src/utilities/createInitialPermissions.js

```diff
@@ -51,6 +51,7 @@ const permissionsRoles = [
       'changeUserStatus',
       'changeUserRehireableStatus',
       'updatePassword',
+      'resetPassword',
```

Comment:
```suggestion
```
I don't think this is absolutely necessary for Admins. Owners could still grant it to Admins without this here. This will re-add it to admins on every startup which could be annoying if unwanted.